controllers/usersController.ts

```diff
@@ -68,6 +72,67 @@ export const checkDuplicateEmail: RequestHandler = async (req, res, next) => {
     next();
 };
 
-export const loginUser: RequestHandler = (req, res) => {
-    // Find user, get hashed password and use bcrypt to compare. We use sessions database to store user sesssions?
+export const loginUser: RequestHandler = async (
+    req: Request,
+    res: Response
+) => {
+    const { emailOrUsername, password } = req.body;
+
+    type User = {
```

Comment:
i thought you told me we can the User type from prisma?

Reply:
I actually haven't found a way to to do it. The type that I found that's generated by Prisma is Prisma.userSelect which is the type for the value of select when doing queries. It's like this

![image](https://user-images.githubusercontent.com/79148590/216864716-b38cf082-f193-435e-ba7a-180a8f68e6f1.png)

Will read up on the docs and update

Reply:
used Prisma.userCreateInput for the return types since I didn't select which fields to return.